### ISSUE TITLE
Report does not work

### ISSUE THREAD

**Issue by hugo-akaora** [ticgal/actualtime#138]
Hello,

Thank you for your plugin!

Reports seem to do not work : 
https://github.com/ticgal/actualtime/blob/develop/report/actualtimetotal/actualtimetotal.php -> `   RIGHT JOIN glpi_tickettasks ON glpi_tickettasks.id = glpi_plugin_actualtime_tasks.tasks_id` should be replaced by `RIGHT JOIN glpi_tickettasks ON glpi_tickettasks.id = glpi_plugin_actualtime_tasks.==id==`

https://github.com/ticgal/actualtime/blob/develop/report/actualtimetotal/actualtimeuser.php crashes (blank page) with PHP error in log : 
```
[2024-09-03 10:47:12] glpiphplog.CRITICAL:   *** Uncaught Exception Error: Call to undefined function _() in /volume4/web/glpi/plugins/actualtime/report/actualtimeuser/actualtimeuser.php at line 51
  Backtrace :
  public/index.php:82                                require()
```

Maybe the core plugin was updated but not reports related to?

PS: sorry, I don't have enough time and knowledge to make PR.

Thank you

**Comment by OscarBeiro** [comment]
I don't think we will fix this issue since we no longer use **reports** plugin.
Sorry for the inconvenience
